pkg/flowaggregator/flowaggregator.go

```diff
@@ -432,11 +435,14 @@ func (fa *flowAggregator) sendFlowKeyRecord(key ipfixintermediate.FlowKey, recor
 		fa.fillK8sMetadata(key, record.Record)
 		fa.aggregationProcess.SetCorrelatedFieldsFilled(record)
 	}
-	if !fa.aggregationProcess.AreExternalFieldsFilled(*record) {
-		fa.fillPodLabels(key, record.Record)
+	if fa.includePodLabels {
+		if !fa.aggregationProcess.AreExternalFieldsFilled(*record) {
+			fa.fillPodLabels(key, record.Record)
+			fa.aggregationProcess.SetExternalFieldsFilled(record)
+		}
+	} else {
 		fa.aggregationProcess.SetExternalFieldsFilled(record)
```

Comment:
This line should be inside the if statement.

Reply:
Thanks for the review! It seems that in the if statement, we have really ensured the external fields to be filled. I added line 444 for the case  that pod labels are not included. I tried to mark the external fields as filled to keep consistence with the case pod labels are provided. Should I just remove the line? Correct me if my understanding is wrong!

Reply:
Got it, missed you have this line inside the if statement, you could remove this else statement.

Reply:
done